### PR TITLE
master node cordon code update for perf run

### PR DESCRIPTION
**Pull request by piyushgupta1551**
- Added a new generic perf role - `perf-cluster-tasks` which can be used in the future to add more plays whenever needed for perf runs
- Added 1 play to cordon master node(s)
- Added new install-k8s-perf.yml file as per suggestion with new role inside it.

### CODE REVIEW

File: roles/perf-cluster-tasks/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- name: cordon master node
+  command: "kubectl cordon `echo $HOSTNAME`"
```
Comment (mkumatag) [inline roles/perf-cluster-tasks/tasks/main.yml:2]: at times, nodes will have a different name than hostname, can we cordon all the master nodes instead.

hints:

list master nodes:
```
kubectl get node --selector='node-role.kubernetes.io/master'
```
you can also use xargs here

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
you can remove empty `defaults/main.yml` file

File: roles/perf-cluster-tasks/tasks/main.yml (hunk 1 above)
Comment (piyushgupta1551) [reply roles/perf-cluster-tasks/tasks/main.yml]: Changed to `kubectl get node --selector='node-role.kubernetes.io/master' --no-headers | awk '{print $1}' | xargs kubectl cordon`

**Comment by piyushgupta1551** [comment]
@mkumatag - Both changes done and rebased.

**Review by Rajalakshmi-Girish — APPROVED** [review]
lgtm

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**